src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/Attendance.java

```diff
@@ -23,26 +23,25 @@
 @AllArgsConstructor
 @Entity
 @Table(name = "OFFENDER_COURSE_ATTENDANCES")
-@ToString(of = {"eventId", "offenderBookingId"})
+@ToString(of = {"eventId", "offenderBooking"})
```

Comment:
actually this doesnt look right, the toString method supplies a printable string listing important fields. can you change it to   offenderBooking.id or something similar?